packages/venia-ui/lib/components/ProductOptions/swatch.css

```diff
@@ -1,6 +1,6 @@
 .root {
     composes: root from './tile.css';
-    background-color: rgb(var(--venia-swatch-bg));
+    background: var(--venia-swatch-bg);
```

Comment:
The `background` property is shorthand and allows us to use a single variable to provide either the URL or the color.